docs/docs/developers/sandbox/references/sandbox-reference.md

```diff
@@ -2,57 +2,11 @@
 title: Sandbox Reference
 ---
 
-Here you will find a reference to everything available within the Sandbox.
+:::tip
 
-## Installation
+Follow the [quick start guide](../../getting_started/quickstart.md) to install the sandbox.
```

Comment:
Well that would make us have to update in two places. We should never have to repeat ourselves, that leads to "more" and we always want "less" 😆 

But since we're leaving only one "version" in the quickstart, I can move the other ones here (and link it)